packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -211,11 +211,11 @@ Additionally, the property `enableSmsRole` can be used to override the CDK's def
 suppress automatic role creation.
 
 ```ts
-import { Role } from '@aws-cdk/aws-iam';
-
-const poolSmsRole = new Role(this, 'userpoolsmsrole', { /* ... */ });
+const poolSmsRole = new iam.Role(this, 'userpoolsmsrole', {
+  assumedBy: new iam.ServicePrincipal('foo'),
+});
 
-new UserPool(this, 'myuserpool', {
+new cognito.UserPool(this, 'myuserpool', {
```

Comment:
Not a big fan of prefixing `cognito` everywhere. Would've preferred if the fixture had individually imported what's required, so the README stays clean.

OTOH doing so would mean that any new type being used here would have the fixture be updated. Given that we have no validation for this, maybe this is ok.